Project-Aurora/Project-Aurora/Project-Aurora.csproj

```diff
@@ -98,17 +98,18 @@
     <PackageReference Include="MouseKeyHook.NetCore" Version="5.6.1.1" />
     <PackageReference Include="NAudio" Version="2.1.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="NLog" Version="4.7.10" />
-    <PackageReference Include="Octokit" Version="2.0.1" />
-    <PackageReference Include="PropertyChanged.Fody" Version="4.1.0" />
+    <PackageReference Include="NLog" Version="5.0.4" />
```

Comment:
I can't see any logs created :)

Reply:
I'll just downgrade instead of trying to fix in this PR.

Reply:
This was my solution to trying to upgrade these outdated libraries :)